app/src/docs/v1.api-spec.yaml

```diff
@@ -4155,20 +4417,51 @@ components:
               description: >-
                 The original file type of the `content` document template.
               example: txt
+    TimeStampCreate:
+      type: object
+      properties:
+        createdBy:
+          type: string
+          description: >
+            The username of the person who created the object.
+          example: bsmith@idir
+        createdAt:
+          type: string
+          description: >
+            The UTC date and time when the object was created.
+          example: '2020-06-04T18:49:20.672Z'
+        updatedBy:
+          type: string
+          description: >
+            The username of the person who last modified the object.
+          example:
+        updatedAt:
+          type: string
+          description: >
+            The UTC date and time when the object was last modified.
+          example: '2020-06-04T18:49:20.672Z'
     TimeStampUserData:
       type: object
       properties:
         createdBy:
           type: string
+          description: >
+            The username of the person who created the object.
```

Comment:
We should always be using descriptions. Once these are all set up we can configure the extension to enforce it.